DigitalLearningSolutions.Web/Views/VerifyEmail/Index.cshtml

```diff
@@ -0,0 +1,130 @@
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.ViewModels.VerifyEmail
+@model VerifyEmailViewModel
+
+@{
+  ViewData["Title"] = "Verify your email address";
+}
+
+@section NavMenuItems {
+  <partial name="_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Verify your email address</h1>
+
+    @if (Model.UnverifiedEmailsCount == 0) {
+      <p class="nhsuk-body-m"> All your email addresses are verified.</p>
+    } else {
+      var centreEmailsExcludingFirstParagraph = (Model.PrimaryEmail == null
+        ? Model.CentreSpecificEmails.Skip(1)
+        : Model.CentreSpecificEmails).ToList();
+      const string centreEmailExplanation = "You will not be able to access that account until you verify the address.";
```

Comment:
I agree that this is pretty messy, but I'm not 100% sure how best to tidy it up.
I think lets start by making these model properties, so that we can pull apart the later logic into partials more easily.